tests/console/verify_bcache_partitioning.pm

```diff
@@ -0,0 +1,55 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: Validate bcache in writethrough
+# The backing device is /home and it is attached to the caching device which is the second
```

Comment:
it is better described in the ticket: `Bcache uses not formatted native linux partition as backing device and second disk as a whole as caching device.` in fact /home is not the backing device, is one of the mount point for the cached device. The backing device is in the 4th partition of vda.

Reply:
this looks to me like a description of the bcache rather of the test scenario

Reply:
sure, it is a definition, but when you tried to be specific mentioned /home as a backing device, do you think that is correct? is the partition, right?

Reply:
i think it is correct. I try to describe what the job is intended to do. So /home is used as a backing device (/dev/vda) with the second disk as caching device (/dev/vdb) as probably would be in a really setup which uses bcache. i guess the description could be better so i am open for suggestions.

Reply:
the backing device is a specific partition in the first disk (/dev/vda3?), which /home is its mount point and regarding the cache device I'd rephrase it to "the whole second disk" as well (meaning no partitions).

Reply:
2ffcdfc00